CorePayments/src/main/java/com/paypal/android/corepayments/api/FetchClientToken.kt

```diff
@@ -0,0 +1,44 @@
+package com.paypal.android.corepayments.api
+
+import androidx.annotation.RestrictTo
+import com.paypal.android.corepayments.APIClientError
+import com.paypal.android.corepayments.APIRequest
+import com.paypal.android.corepayments.CoreConfig
+import com.paypal.android.corepayments.HttpMethod
+import com.paypal.android.corepayments.RestClient
+import com.paypal.android.corepayments.base64encoded
+import org.json.JSONObject
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class FetchClientToken(
```

Comment:
```suggestion
class AuthenticationSecureTokenServiceAPI(
```
This name would fit our naming convention if we want to be 100% consistent with the other `API` suffixed classes.